docs/installation.rst

```diff
@@ -0,0 +1,24 @@
+
+Installation
+============
+
+**Starting from scratch**
+
+----
+
+Python and associated packages for science are freely available. Convenient science python package setups are available from `Enthought <https://store.enthought.com>`_ and `Continuum Analytics <http://continuum.io/downloads>`_. Enthought also includes an IDE, though there are a number of choices. Core science packages such as numpy, scipy, matplotlib, pandas and many others may also be installed directly via pip or your favorite package manager.
```

Comment:
We should deprecate guiding to enthought. I think they are getting out of the python distribution business.

Reply:
From https://assets.enthought.com/downloads/, 

The Canopy GUI is at end of life. Canopy installers are no longer publicly downloadable.

It is a bummer.